runtime/sound/src/soundinstance.cpp

```diff
@@ -855,8 +855,15 @@ LTRESULT CSoundInstance::StartRendering( )
 		else
 		{
 			dwCurPos = m_dwDuration - m_dwTimer;
+#ifdef USE_OPENAL
+			//need to set position after playing, since OpenAL sources are zero
+			//until then
+			GetSoundSys()->PauseStream( m_hStream, 0 );
+			GetSoundSys()->SetStreamMsPosition( m_hStream, dwCurPos );
+#else
 			GetSoundSys()->SetStreamMsPosition( m_hStream, dwCurPos );
```

Comment:
does DirectX DSOUND8 need this order or will it work with the openal order?

if we can do pause stream and then set position for both, then we could do away with the ifdef